underfs/hdfs/src/main/java/alluxio/underfs/hdfs/acl/SupportedHdfsAclProvider.java

```diff
@@ -67,18 +70,80 @@ public AccessControlList getAcl(FileSystem hdfs, String path) throws IOException
     return acl;
   }
 
-  private AclEntryType getAclEntryType(AclEntry entry) throws IOException {
+  @Override
+  public void setAcl(FileSystem hdfs, String path, AccessControlList acl) throws IOException {
+    // convert AccessControlList into hdfsAcl
+    List<AclEntry> aclSpecs = new ArrayList<>();
+
+    for (alluxio.security.authorization.AclEntry entry : acl.getEntries()) {
+      AclEntry hdfsAclEntry = getHdfsAclEntry(entry);
+      aclSpecs.add(hdfsAclEntry);
+    }
+    // set hdfsAcl;
+    try {
+      hdfs.setAcl(new Path(path), aclSpecs);
+    } catch (UnsupportedOperationException e) {
+      LOG.warn("Setting ACL on HDFS with acl disabled");
```

Comment:
this logging may be spammy, so we should not log this.

Reply:
done